templates/terraform/examples/ml_model_basic.tf.erb

```diff
@@ -0,0 +1,5 @@
+resource "google_ml_engine_model" "default" {
```

Comment:
Can we also get a ml_model_full example that sets as many of the fields as possible?